static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -404,24 +406,28 @@ define([
                 adUnit         = adUnitOverride ?
                     ['/', config.page.dfpAccountId, '/', adUnitOverride].join('') : config.page.adUnit,
                 id             = $adSlot.attr('id'),
-                sizeMapping    = defineSlotSizes($adSlot),
+                slot,
+                size,
+                sizeMapping;
+
+            if ($adSlot.data('out-of-page')) {
+                slot = googletag.defineOutOfPageSlot(adUnit, id);
+            } else if ($adSlot.data('fluid') && cookies.get('adtest') === 'tm2') {
```

Comment:
We're not going to have adtest cookies on for the most part in production - is this just for internal demonstrations? Is this actually going to be released to production?